wire-ios-data-model/Source/Model/Message/ZMMessage+Reaction.swift

```diff
@@ -24,6 +24,7 @@ extension ZMMessage {
         reaction: WireProtos.Reaction,
         senderID: UUID,
         conversation: ZMConversation,
+        creationDate: Date?,
```

Comment:
should it be optional? Do we know a case when it can be nil?

Reply:
`updateEvent.timestamp` is optional